var/spack/repos/builtin/packages/ncl/package.py

```diff
@@ -293,6 +293,17 @@ def prepare_install_config(self):
         with open(config_answers_filename, "r") as f:
             config_script(input=f)
 
+        # Modify Site.local to add RPC library if HDF4 is used w/external XDR
+        if "+hdf4" in self.spec:
+            hdf4 = self.spec["hdf"]
+
+            if "+external-xdr" in hdf4 and hdf4["rpc"].name != "libc":
```

Comment:
Hi @alalazo - that will not work because we need to check for the absence of libc, not its presence. I could change it to:

```
if self.spec.satisfies("^hdf4+external-xdr") and not self.spec.satisfies("^libc"):
```

but I'd still rather check for it in the hdf(4) spec because I worry that libc could creep into the NCL spec via some other dependency.

If that sounds reasonable, I'll push to my branch a small syntax change to use `satisfies` and volunteer to maintain the package.

Reply:
Ok, then what about:
```
if self.spec.satisfies("^hdf4+external-xdr") and not self.spec["hdf4"].satisfies("^libc"):
```
? Great if you want to maintain the package!

Reply:
@alalazo - sure, no problem. I've made the suggested change and pushed to my fork.